comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Comment(TimestampMixin, models.Model):
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    recipe = models.ForeignKey('recipes.Recipe', on_delete=models.CASCADE)
```

Comment:
good job on this, using the string instead of importing and using the model class. This way is preferred because it avoids issues when Django is starting up and trying to instantiate model classes _and_ create relationships at the same time.